mapcompose/src/main/java/ovh/plrapps/mapcompose/api/PathApi.kt

```diff
@@ -37,6 +38,7 @@ fun MapState.addPath(
     pathData: PathData,
     width: Dp? = null,
     color: Color? = null,
+    fillColor: Color? = color,
```

Comment:
I would prefer to have `null` value as default for `fillColor` (here and at other relevant places), and apply it when a non-null value set.
This way, we don't need a "isClosePath" flag.
That would eliminate an issue with `isClosePath`: if we change the value (from e.g `true` to `false`), the path won't be redrawn since `isClosePath` isn't backed by a state.

Reply:
ok - so basically when (and only when) there is a fill-color set we would `close()` the path then?

Reply:
i believe so, yes.